### ISSUE TITLE
Converter to slideshows and presentations

### ISSUE THREAD

**Issue by rbeezer** [rbeezer/mathbook#856]
Write an XSLT converter that takes PTX structured with `slideeshow` and 'frame` and outputs a presentation (ala PowerPoint, etc).

Output could be Beamer, S5, Sozi, or ...

I have a crude experimental version, so be in touch if you want something to start with.

**Comment by rbeezer** [comment]
reveal.js looks like a great format to target for output.  https://revealjs.com/#/

**Comment by kcrisman** [comment]
> reveal.js looks like a great format to target for output.
> https://revealjs.com/#/
>
>
Yes.  RStudio has some similar functionality for its worksheets, FYI - see
https://rstudio.github.io/learnr/ though I don't know if that would be
immediately applicable to us.  But at least it confirms your idea.

**Comment by kcrisman** [comment]
Should this be closed, since we now have the slideshow target (and beamer, for that matter)?  That doesn't mean it's perfect yet, but I have to say it already is working well enough for me to use it this semester.  (Expect PRs if so.)

**Comment by rbeezer** [comment]
Two such converters now exist: reveal.js for HTML (`xsl/pretext-revealjs.xsl`, recently rebuilt on reveal.js 6 with embedded-resource and single-file options — #3058, #3060, #3061) and Beamer for LaTeX (`xsl/pretext-beamer.xsl`, #3012).  Closing as implemented.

*Claude Fable 5, acting as a coding assistant for Rob Beezer*
